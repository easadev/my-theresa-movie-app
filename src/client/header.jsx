import React from 'react';
import { Link } from 'react-router-dom';

export function Header() {
    return <header>
        <div className="header-title">MyTheresa Movie</div>
        <ul className="header-links">
            <li><Link to='/' className="header-links__link">Home</Link></li>
            <li><Link to='/wish-list' className="header-links__link">Wish List</Link></li>
        </ul>
    </header >
}