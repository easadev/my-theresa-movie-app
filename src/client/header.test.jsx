import { create } from 'react-test-renderer';
import React from 'react';
import { Header } from './header';
import { MemoryRouter } from 'react-router-dom';

it('Tests Header component mounts fine', () => {
    expect(create(<>
    <MemoryRouter >
        <Header />
    </MemoryRouter>
    </>).toJSON()).toMatchSnapshot();
});