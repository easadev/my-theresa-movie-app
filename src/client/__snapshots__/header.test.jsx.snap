// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests Header component mounts fine 1`] = `
<header>
  <div
    className="header-title"
  >
    MyTheresa Movie
  </div>
  <ul
    className="header-links"
  >
    <li>
      <a
        className="header-links__link"
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      <a
        className="header-links__link"
        href="/wish-list"
        onClick={[Function]}
      >
        Wish List
      </a>
    </li>
  </ul>
</header>
`;
